src/store/index.ts

```diff
@@ -1,7 +1,8 @@
 import {StoreOptions} from 'vuex'
 import {IFlowsState, store as flow} from './flow'
 import {IBuilderState, store as builder} from './builder'
-import trees from './trees/trees'
+// @ts-ignore
+import trees from '@/store/trees/trees.js'
```

Comment:
Can we try and avoid the ts-ignore? We should try not to introduce new ones

Reply:
What do you mean here exactly @jcbashdown ?